WordPress/Docs/PHP/DontExtractStandard.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Don't extract"
```

Comment:
I know that the sniff is called `DontExtract`, but maybe the title can be `Don't use extract()`? To be a bit more clear?